.idea/jarRepositories.xml

```diff
@@ -21,5 +21,10 @@
       <option name="name" value="MavenRepo" />
       <option name="url" value="https://repo.maven.apache.org/maven2/" />
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="maven" />
+      <option name="name" value="maven" />
```

Comment:
Warum führst du jetzt maven ein?

Reply:
Gradle uses maven repos to pull dependencies, in this case a custom repo. We still dont use the program maven though :D